packages/story-editor/src/components/errorBoundary/index.js

```diff
@@ -53,7 +53,6 @@ class ErrorBoundary extends Component {
 }
 
 const shouldDisableErrorBoundaries =
-  typeof WEB_STORIES_DISABLE_ERROR_BOUNDARIES !== 'undefined' &&
```

Comment:
Ignore changes here, this is reverting a PR that causes multiple test failures, see https://github.com/GoogleForCreators/web-stories-wp/pull/10589#issuecomment-1042875858